src/UI/Components/OldPaginationWidget/Provider.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { Pagination as PaginationComponent } from "@patternfly/react-core";
+import styled from "styled-components";
+import { PageSize, Pagination, RemoteData } from "@/Core";
+import { PaginationPageSizes } from "@/Core/Domain/PageSize";
+import { CurrentPage } from "@/Data/Common/UrlState/useUrlStateWithCurrentPage";
+
+type Data = RemoteData.Type<
+  string,
+  {
+    handlers: Pagination.Handlers;
+    metadata: Pagination.Metadata;
+  }
+>;
+
+interface Props {
+  data: Data;
+  pageSize: PageSize.Type;
+  setPageSize: (size: PageSize.Type) => void;
+  setCurrentPage: (currentPage: CurrentPage) => void;
+  variant?: "top" | "bottom";
+}
+
+/**
+ * Wrapper for Pagination for V1 queries as we are basing our-selfs on links served by backend - This component is Temporarily added to maintain pagination functionality for V1 queries until all will be removed
+ *
+ * Note: Parameters responsible for pagination on endpoint doesn't allow to pass numerical range for displayed results, or any other way to navigate through pages other than previous/next page
+ */
+export const Provider: React.FC<Props> = ({
```

Comment:
Maybe we can avoid renaming it in the index file, and add a extra note about his being the old one?

Reply:
removed the renmaing, there is note in the docstrings about it being used in V1 and should be removed when all v1 will be gone